### PR TITLE
[Holy Paladin] Initial The War Within port and Guide v1

### PR DESCRIPTION
**Pull request by Texleretour**
### Description

- Add Initial support for 11.0.2 and onwards.
- Change from Unmaintained to MaintainedPartially
- Change default vue to guide
- Guide features: 
  - Holy Shock, Holy Prism and Beacon of Virtue cast efficiencies
  - Holy Power chart w/ waste
  - Beacon of Light/Faith uptimes
  - All Healing Cooldowns  cast efficiencies
  - Blessing of Seasons cast efficiencies
  - Explanations

This guide is not complete, but I think it's a good start. More on that in my issue #6959.

### Testing

- Test report URL:
  -  `/report/fHk1awXpvV3yMZg7/59-Mythic+Rashok,+the+Elder+-+Kill+(2:50)/Priiyaa/standard`
  - `/report/HFjyzAZxgRPh4TbY/1-Normal+Raszageth+the+Storm-Eater+-+Kill+(9:26)/4-Amenadiell/standard`
- Screenshot(s):
image
image
image

### CODE REVIEW

File: src/analysis/retail/paladin/holy/modules/talents/BlessingOfTheSeasons.tsx [hunk 1]
```diff
@@ -35,4 +41,50 @@ export class BlessingOfTheSeasons extends Analyzer {
...
+
+  get guideSubsection(): JSX.Element {
+    const explanation = (
+      <p>
+        <b>
+          <SpellLink spell={TALENTS.BLESSING_OF_SUMMER_TALENT} />
+        </b>{' '}
+        is a unique ability that cycles through 4 different buffs.{' '}
+        <SpellLink spell={TALENTS.BLESSING_OF_SUMMER_TALENT} /> is the most powerful one and
+        converts healing into damage. Always use it on yourself then slam healing (overhealing
```
Comment (mfield) [inline src/analysis/retail/paladin/holy/modules/talents/BlessingOfTheSeasons.tsx]: it's also a strong healing throughput cd now.

File: src/analysis/retail/paladin/holy/modules/talents/HolyShock.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        </RoundedPanel>
+      </div>
+    );
+
+    return explanationAndDataSubsection(explanation, data, GUIDE_CORE_EXPLANATION_PERCENT);
+  }
+
+  subStatistic() {
+    return (
+      <CastEfficiencyBar
```
Comment (mfield) [inline src/analysis/retail/paladin/holy/modules/talents/HolyShock.tsx:55]: cdr talents have modules already, but need updating:
- crusader's might: currently 1500 ms, should be 2000
- imbued infusions: currently 2000 ms, should be 1000

**Review by mfield — COMMENTED** [review]
looks like a good start

File: src/analysis/retail/paladin/holy/modules/talents/HolyShock.tsx (hunk 2 above)
Comment (Texleretour) [reply src/analysis/retail/paladin/holy/modules/talents/HolyShock.tsx:55]: thanks for the review!

**Review by emallson — APPROVED** [review]

**Pull request merged**